src/tools/rbd/ArgumentTypes.cc

```diff
@@ -270,7 +270,7 @@ void add_create_image_options(po::options_description *opt,
     (IMAGE_FEATURES.c_str(), po::value<ImageFeatures>()->composing(),
      ("image features\n" + get_short_features_help(true)).c_str())
     (IMAGE_SHARED.c_str(), po::bool_switch(), "shared image")
-    (IMAGE_STRIPE_UNIT.c_str(), po::value<uint64_t>(), "stripe unit")
+    (IMAGE_STRIPE_UNIT.c_str(), po::value<ImageObjectSize>(), "stripe unit")
```

Comment:
Nit: can you update the description to document that it accepts B/K/M (like object size above)? You will need to update the cli test due to the help test change.